src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -138,7 +138,11 @@ void NotationViewInputController::onNotationChanged()
         m_view->hideElementPopup();
 
         if (AbstractElementPopupModel::supportsPopup(type)) {
-            m_view->showElementPopup(type, selectedItem->canvasBoundingRect());
+            if (selectedItem->isDynamic()) {
+                m_view->showElementPopup(type, toDynamic(selectedItem)->adjustedBoundingRect());
```

Comment:
The same doubt again about `adjustedBoundingRect`. You could decide to move all changes related to the addition of this method into yet another separate PR, so that the initial popup functionality PR will be easy to merge.